### DISCUSSION TITLE
部署到服务器的项目文件，未压缩，请问怎么处理？

### DISCUSSION THREAD

**Discussion by Lands-1203** [umijs/umi#11267] [Q&A]
将umi4的项目部署到服务器上后，没有做域名映射，通过IP访问，访问到的umi.13465.js文件未压缩，请问这是配置问题，还是说由于我是IP访问项目认为我是开发环境？还是说我需要再linux配置什么？

**Comment by fz6m** [comment]
要把产物构建出来再部署到服务器的 nginx 上，umi 打包出来的结果是一个 SPA 静态站点，不需要启动服务。

**Reply by Lands-1203** [reply]
嗯嗯 是将dist部署当nginx上的 很奇怪，发现这个问题后重新打包部署一次后就发现压缩了的
